draft-ietf-dnsop-svcb-https.md

```diff
@@ -596,20 +596,22 @@ Providing the proxy with the final SvcDomainName has several benefits:
 ## Authoritative servers
 
 When replying to a SVCB query, authoritative DNS servers SHOULD return
-A, AAAA, and SVCB records (as well as any relevant CNAME or
+A, AAAA, and SVCB records (as well as following any relevant CNAME or
 {{!DNAME=RFC6672}} records) in the Additional Section for any
```

Comment:
Including useful records in the additional section, is called additional section processing in RFC1035.
When comparing SVCB with MX and SRV, note that MX and SRV may not refer to an alias. This is documented in https://tools.ietf.org/html/rfc2181#section-10.3 and in https://tools.ietf.org/html/rfc2782 on page 4, quoting: "The name MUST NOT be an alias".

Reply:
I suggest to remove the text in parenthesis.

Reply:
OK, removed.  CNAMEs within the bailiwick would be a strange configuration anyway.